### ISSUE TITLE
Add parameter for peptide inference

### ISSUE THREAD

**Issue by veitveit** [ProtProtocols/IsoLabeledProtocol#45]
Simple options would be:
a) grouping of shared proteins (this is happening now)
b) only unique peptide

Future options could/will be different versions of the parsimony principle

**Comment by jgriss** [comment]
@veitveit 

Does this apply to how *peptides* are grouped or how *protein inference* is done?

**Comment by veitveit** [comment]
Grouping

**Comment by jgriss** [comment]
@veitveit 
I've added the GUI element to my local working copy but I'm struggling to implement the actual functionality to only use unique peptides.

Currently, the grouping is done by peptide shaker. The protein groups are automatically generated so in the MSnbase object all our peptides have nprot = 1 (meaning a unique matching).

**Comment by jgriss** [comment]
@veitveit 

I think that it will require a bit more code since MSnbase is currently treating a protein group as one protein. But I fully agree that this is doable and will be a very sensible feature. I'll add the GUI element just without any option for now.

**Comment by jgriss** [comment]
@veitveit The drop-down box is there, but currently only displays a single option. Do you want to keep this issue open until we have the functionality in the R code as well?

**Comment by veitveit** [comment]
@jgriss  yes, let's leave it open until we have a first solution for an alternative grouping
